src/dashboard/apps/consent/tests/test_admin.py

```diff
@@ -29,3 +33,59 @@ def test_admin_manager_name(rf):
     # The manager name must be the default Django manager: `objects`.
     manager_name = queryset.model._default_manager.name
     assert manager_name == "objects"
+
+
+@pytest.mark.django_db
+def test_make_revoked_action(client, patch_timezone_now):
+    """Tests the 'make_revoked' action for ConsentAdmin."""
+    # Initialize admin user
+    admin_user = AdminUserFactory()
+
+    # create a consent
+    assert Consent.objects.count() == 0
+    DeliveryPointFactory()
+    assert Consent.objects.count() == 1
+
+    # Post action with selected consent
+    consent = Consent.objects.first()
+    data = {
+        "action": "make_revoked",
+        "_selected_action": [
+            consent.id,
+        ],
+    }
+    client.force_login(admin_user)
+    client.post(reverse("admin:qcd_consent_consent_changelist"), data)
+
+    consent.refresh_from_db()
+    assert consent.status == REVOKED
```

Comment:
To be explicit, I think it would be great to also assert the status of the consent before performing the admin action.